Add URL datepicker (#865)
contao/docs: docs/dev/reference/dca/fields.md

| cols               | Columns (`integer`)              | Number of columns (used for `textarea`, `radioTable` and `tableWizard` fields).                                                                                                                                |
| csv                | Delimiter (`string`)             | The choice of this field will not be stored as serialized string but rather as given delimiter-separated list. Example: `'eval' => ['csv'=>',']`                 |
| customRgxp        | Regular expression (`string`) | {{< version "4.11" >}} Custom regular expression to be used when using `'rgxp' => 'custom'` |

> | datepicker         | true/false (`bool`)           | If true the current field has a date picker [MooTools-DatePicker](https://github.com/arian/mootools-datepicker).                                                                                                                             |

Reviewer: ```suggestion
| datepicker         | true/false (`bool`)           | If true the current field will have a [MooTools-DatePicker](https://github.com/arian/mootools-datepicker).                                                                                                                             |
```
